### PR TITLE
Troubleshoot failing `macos` check on CI

### PR DESCRIPTION
**Pull request by jtpio**
## References

Continuation of #1190 

## Code changes

CI fixes.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by jtpio** [comment]
OK so all the tests are now failing, this looks new...

But the macos install check now passes:

image

With the correct Python version now being used across steps:

image

File: .github/workflows/packaging.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@ jobs:
           path: ./dist
       - name: Install the prerequisites
         run: |
-          ${{ matrix.py_cmd }} -m pip install pip wheel
+          ${{ matrix.py_cmd }} -m pip install pip wheel jupyterlab~=3.0 notebook~=6.4
```
Comment (jtpio) [inline .github/workflows/packaging.yml:85]: Not sure how the extension checks could have worked before without these packages :thinking:

**Comment by jtpio** [comment]
Marking as ready as this at least fixes the macos install check.

The other failures still need a fix too but they look unrelated to this change: https://github.com/voila-dashboards/voila/pull/1192#issuecomment-1228433656

**Comment by trungleduc** [comment]
Thanks @jtpio !

**Pull request merged**